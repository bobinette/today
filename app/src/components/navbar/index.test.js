import React from 'react';
import { fromJS } from 'immutable';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { NavBar } from '.';

configure({ adapter: new Adapter() });

test('render the log list', () => {
  const fetchLogs = jest.fn();
  const props = {};

  const tree = shallow(<NavBar {...props} />);
  expect(tree).toMatchSnapshot();
});
