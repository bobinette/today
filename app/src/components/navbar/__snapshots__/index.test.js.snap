// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render navbar 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="row w-100 flex flex-align-items-center flex-space-between"
        >
          <h5
            className="NavBar__AppName fg-white"
          >
            <img
              alt=""
              className="TodayLogo"
              height="24"
              src="/assets/leaf-inverse.png"
              width="24"
            />
            Today
          </h5>
          <div
            className="GithubLogo"
          >
            <a
              className="GithubLink"
              href="https://github.com/bobinette/today"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-github"
              />
            </a>
          </div>
        </div>
      </div>,
      "className": "navbar navbar-row navbar-top navbar-inverse bg-primary TodayNavbar",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row w-100 flex flex-align-items-center flex-space-between"
        >
          <h5
            className="NavBar__AppName fg-white"
          >
            <img
              alt=""
              className="TodayLogo"
              height="24"
              src="/assets/leaf-inverse.png"
              width="24"
            />
            Today
          </h5>
          <div
            className="GithubLogo"
          >
            <a
              className="GithubLink"
              href="https://github.com/bobinette/today"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-github"
              />
            </a>
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5
              className="NavBar__AppName fg-white"
            >
              <img
                alt=""
                className="TodayLogo"
                height="24"
                src="/assets/leaf-inverse.png"
                width="24"
              />
              Today
            </h5>,
            <div
              className="GithubLogo"
            >
              <a
                className="GithubLink"
                href="https://github.com/bobinette/today"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-github"
                />
              </a>
            </div>,
          ],
          "className": "row w-100 flex flex-align-items-center flex-space-between",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt=""
                  className="TodayLogo"
                  height="24"
                  src="/assets/leaf-inverse.png"
                  width="24"
                />,
                "Today",
              ],
              "className": "NavBar__AppName fg-white",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "TodayLogo",
                  "height": "24",
                  "src": "/assets/leaf-inverse.png",
                  "width": "24",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "Today",
            ],
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                className="GithubLink"
                href="https://github.com/bobinette/today"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-github"
                />
              </a>,
              "className": "GithubLogo",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fab fa-github"
                />,
                "className": "GithubLink",
                "href": "https://github.com/bobinette/today",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fab fa-github",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "a",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row w-100 flex flex-align-items-center flex-space-between"
          >
            <h5
              className="NavBar__AppName fg-white"
            >
              <img
                alt=""
                className="TodayLogo"
                height="24"
                src="/assets/leaf-inverse.png"
                width="24"
              />
              Today
            </h5>
            <div
              className="GithubLogo"
            >
              <a
                className="GithubLink"
                href="https://github.com/bobinette/today"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-github"
                />
              </a>
            </div>
          </div>
        </div>,
        "className": "navbar navbar-row navbar-top navbar-inverse bg-primary TodayNavbar",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row w-100 flex flex-align-items-center flex-space-between"
          >
            <h5
              className="NavBar__AppName fg-white"
            >
              <img
                alt=""
                className="TodayLogo"
                height="24"
                src="/assets/leaf-inverse.png"
                width="24"
              />
              Today
            </h5>
            <div
              className="GithubLogo"
            >
              <a
                className="GithubLink"
                href="https://github.com/bobinette/today"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-github"
                />
              </a>
            </div>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5
                className="NavBar__AppName fg-white"
              >
                <img
                  alt=""
                  className="TodayLogo"
                  height="24"
                  src="/assets/leaf-inverse.png"
                  width="24"
                />
                Today
              </h5>,
              <div
                className="GithubLogo"
              >
                <a
                  className="GithubLink"
                  href="https://github.com/bobinette/today"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-github"
                  />
                </a>
              </div>,
            ],
            "className": "row w-100 flex flex-align-items-center flex-space-between",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    className="TodayLogo"
                    height="24"
                    src="/assets/leaf-inverse.png"
                    width="24"
                  />,
                  "Today",
                ],
                "className": "NavBar__AppName fg-white",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "TodayLogo",
                    "height": "24",
                    "src": "/assets/leaf-inverse.png",
                    "width": "24",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "Today",
              ],
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="GithubLink"
                  href="https://github.com/bobinette/today"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fab fa-github"
                  />
                </a>,
                "className": "GithubLogo",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="fab fa-github"
                  />,
                  "className": "GithubLink",
                  "href": "https://github.com/bobinette/today",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fab fa-github",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "a",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
